.travis.yml

```diff
@@ -41,10 +41,5 @@ matrix:
   - rvm: 2.1.10
```

Comment:
this should be 2.1.9, since all aio packages for puppet4 pc1 ship with 2.1.9.

when you add puppet 5 support, it will be 2.4.0 for that
https://github.com/puppetlabs/puppetlabs-java/blob/master/.travis.yml#L25